recipes/jsoncpp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.10.6" %}
+
+package:
+  name: jsoncpp
+  version: {{ version }}
+
+source:
+  fn: jsoncpp-{{ version }}.tar.gz
+  url: https://github.com/open-source-parsers/jsoncpp/archive/{{ version }}.tar.gz
+  sha256: 7c285fc40ad0c113e436a1271c4e38b5017b5c7782c306e90be9d6b2ffa90212
+
+build:
+  number: 0
+  skip: true  # [win or (unix and not py27)]
```

Comment:
Added back the Windows skip, but have left everything else Windows related intact to save time.